.gitignore

```diff
@@ -0,0 +1,3 @@
+*.pyc
```

Comment:
These should be in your global .gitignore, rather than the project one.
See: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore